client/my-sites/customer-home/style.scss

```diff
@@ -33,16 +32,29 @@ body.is-section-home.theme-default.color-scheme {
 		border: 1px solid var( --studio-gray-5 );
 		box-sizing: border-box;
 		border-radius: 4px;
-		width: 32px;
+		max-width: 58px;
 		height: auto;
 		margin-bottom: 8px;
+		flex: 1 0 auto;
+	}
+
+	.customer-home__site-domain {
+		display: flex;
+		flex-direction: row;
 
-		@include break-small {
-			width: 58px;
-			height: auto;
+		.customer-home__site-domain-text {
+			white-space: nowrap;
+    		overflow: hidden;
+    		text-overflow: ellipsis;
+			width: calc( 100vw - 120px );
+			max-width: fit-content;
 		}
 	}
 
+	.components-external-link__icon {
+		margin-top: 1px;
+	}
```

Comment:
Rather than reaching into the `<ExternalLink>` component and updating its child elements you could also try centering it within the flex box like this:

```scss
.customer-home__site-domain {
  // You've already got these rules
  display: flex;
  flex-direction: row;

  // Maybe add this one?
  align-items: center;
}
```

Reply:
Thanks for the suggestion. I wasn't quite able to center things as expected with this approach.